dss_datamover/socket_communication.py

```diff
@@ -77,7 +92,8 @@ def connect(self, host, port):
 
         is_connection_refused = True
         connection_time_start = datetime.now()
-        time_to_sleep = CONNECTION_DELAY_INTERVAL
+        time_to_sleep = self.config.get("socket_options", ).get("connection_delay_interval")
```

Comment:
why do you need this code when you are overriding the variable . See below line

Reply:
removed unnecessary line